deployment/templates/metrics-configmap.yaml

```diff
@@ -3,7 +3,10 @@ kind: ConfigMap
 metadata:
   name: exporter-metrics-config-map
 data:
-  metrics: |
```

Comment:
Changing this line will also require changing this line in the parser.go file: https://github.com/NVIDIA/dcgm-exporter/blob/main/pkg/dcgmexporter/parser.go#L166